src/components/Place/PlaceDetails/PlaceDetails.tsx

```diff
@@ -81,6 +107,28 @@ export default React.memo(function PlaceDetails(props: PlaceDetailsProps) {
           <PlaceStats place={place} loading={loading} poi={isPoi} />
         </>
       )}
+      {admin && activeTab === PlaceDetailsTab.ContentModeration && (
+        <div className="place-details__content-moderation-container">
+          <h3>{l("components.place_detail.rate_label")}</h3>
+          <div className="place-details__rating-box-container">
+            <RatingButton
+              rating={SceneContentRating.TEEN}
+              active={rating === SceneContentRating.TEEN}
+              onClick={(e) => onChangeRating(e, SceneContentRating.TEEN)}
```

Comment:
instead of having these multiple `onClick` wouldn't it be better to have `onChangeRating` in `RatingButton`?